tests/integ/modin/frame/test_head_tail.py

```diff
@@ -71,23 +71,3 @@ def test_empty_dataframe(n, empty_snowpark_pandas_df):
         comparator=eval_result_and_query_with_no_join,
         check_column_type=False,
     )
-
-
-@pytest.mark.parametrize(
-    "ops",
-    [
-        lambda df: df.head(),
-        lambda df: df.iloc[1:100],
-        lambda df: df.iloc[1000:100:-1],
-    ],
-)
```

Comment:
why do we remove this test ?

Reply:
since we have lots of iloc cases, I feel we should put it into test_iloc.py